cli/internal/cloudcmd/validators_test.go

```diff
@@ -28,6 +30,7 @@ import (
 )
 
 func TestNewValidator(t *testing.T) {
+	// TODO: Add test case for TDX default RTMRs when we know how to handle default values in the config
```

Comment:
This test doesn't use any default measurements though?
Why not just add a test case for qemu TDX and define a list of measurements with TDX measurement length for this test?